frontend/src/components/dashboard/aliases/AliasList.tsx

```diff
@@ -201,6 +201,11 @@ export const AliasList = (props: Props) => {
       </Localized>
     ) : null;
 
+  const handleOnChange = (value: string) => {
+    // removes Unicode characters found within the range of 0080 to FFFF
```

Comment:
question (non-blocking): how did we pick this range?

Reply:
Most common, non-intrusive range to filter by. I was worried I would filter unicode characters we might need.